ops/manifests/manifest-za.yaml

```diff
@@ -23,6 +23,9 @@ applications:
     DJANGO_LOG_LEVEL: INFO
     # default public site location
     GETGOV_PUBLIC_SITE_URL: https://beta.get.gov
+    # NOTE: This needs to go in manifest-staging
+    # This is here for PR purposes only!
+    IS_PRODUCTION: True
```

Comment:
This will be moved to manifest-stable before merging

Reply:
I think that it's time to move it now.